src/app/make_dockerfiles.ml

```diff
@@ -493,8 +493,8 @@ module Image = struct
           Test.test_dashdashversion "ketrew" "3.1.0+dev";
           ketrew_with_pg;
           (* coclobas --version gives: "0.0.2-dev+<commit-hash-prefix>": *)
-          Test.succeeds "coclobas --version | grep 0.0.2-dev";
-          Test.succeeds "ocamlfind list | grep coclobas | grep 0.0.2-dev";
+          Test.succeeds "coclobas --version | grep 0.0.3-dev";
```

Comment:
I actually like having those hard coded and going through the notice-failure/fix/rebuild process :)
that was the code says exactly which tests had to pass before pushing to `hub.docker.com`